packages/astro/types/fonts.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'astro:assets' {
+	/** Run `astro dev` or `astro sync` to generate high fidelity types */
+	export type FontFamily = string;
```

Comment:
Is there a better way? This should be considered an internal type

Reply:
Can you provide some context? I'm not sure I understand what you're trying to achieve and what's the limitation of the current solution

Reply:
Sorry it was clear for me of course :laughing: 

So what I want is:
- By default, the `family` prop of the `Font` component must be typed as string
- If some fonts are registered, make this prop strictly typed using codegen

This currently works but I'm using `astro:assets` to pass the type. I think this makes `FontFamily` part of the public API but I think it should be considered internal.

So what can we do about it? I guess an easy way would be to add `@internal`

Reply:
We could use `@internal`. How would users use the generated type instead?